javatests/arcs/android/systemhealth/testapp/StorageCore.kt

```diff
@@ -350,6 +366,29 @@ class StorageCore(val context: Context, val lifecycle: Lifecycle) {
         watchdog.monitor()
     }
 
+    @Suppress("UNCHECKED_CAST")
+    private suspend fun setUpCleanerHandle(taskHandle: TaskHandle, settings: Settings) {
+        val handle = taskHandle.handleManager.createHandle(
+            HandleSpec(
+                "CleanerHandle",
+                HandleMode.ReadWrite,
```

Comment:
can this be HandleMode.Write?
Myk is planning some performance improvements for write only handle, so using HandleMode.Write only could result in different performances.

Reply:
Sound good! Will do

Reply:
done